### PR TITLE
[ISSUE #3481] Replace this "Map.containsKey()" with a call to "Map.computeIfAbsent()".

### LINKED ISSUE #3481: [Enhancement] Replace this "Map.containsKey()" with a call to "Map.computeIfAbsent()".
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-webhook/eventmesh-webhook-api/src/main/java/org/apache/eventmesh/webhook/api/Manufacturer.java line 57
analysis and explanation:
It’s a common pattern to test the result of a java.util.Map.get() against null or calling java.util.Map.containsKey() before proceeding with adding or changing the value in the map. However the java.util.Map API offers a significantly better alternative in the form of the computeIfPresent() and computeIfAbsent() methods. Using these instead leads to cleaner and more readable code.

### Describe the solution you'd like

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by erzohrakhan**
Fixes #3481 

### Motivation
Use java.util.Map.computeIfAbsent() method instead of java.util.Map.containsKey()(for adding and changing values in the map). This leads to cleaner and more readable code.

### Modifications
Replaced map.containsKey() with map.computeIfAbsent() method

### Documentation

- Does this pull request introduces a new feature? (yes/no) no

### CODE REVIEW

**Review by Alonexc — DISMISSED** [review]
LGTM

**Review by mxsm — COMMENTED** [review]
Hi @erzohrakhan  Please remove the Searching file

File: eventmesh-webhook/eventmesh-webhook-api/src/main/java/org/apache/eventmesh/webhook/api/Manufacturer.java [hunk 1]
```diff
@@ -54,11 +54,7 @@ public void setManufacturerEventMap(Map<String, List<String>> manufacturerEventM
     }
 
     public List<String> getManufacturerEvents(String manufacturerName) {
-        if (!manufacturerEventMap.containsKey(manufacturerName)) {
-            List<String> m = new ArrayList<>();
-            manufacturerEventMap.put(manufacturerName, m);
-            return m;
-        }
+        manufacturerEventMap.computeIfAbsent(manufacturerName, key -> manufacturerEventMap.put(manufacturerName, new ArrayList<>()));
```
Comment (mytang0) [inline eventmesh-webhook/eventmesh-webhook-api/src/main/java/org/apache/eventmesh/webhook/api/Manufacturer.java:57]: 1. It is recommended to use ConcurrentHashMap
2. The logic can be further simplified

**Review by mytang0 — CHANGES_REQUESTED** [review]
1. It is recommended to use ConcurrentHashMap
2. The logic can be further simplified

**Comment by Pil0tXia** [comment]
@erzohrakhan As this PR has not been updated for some time, the issue related to this Pull Request has already been resolved. It is suggested to have this PR closed. Thanks for your contribution.

**Pull request closed without merging**